core/src/main/java/org/ehcache/EhcacheManager.java

```diff
@@ -168,8 +176,9 @@ void closeEhcache(final String alias, final Ehcache<?, ?> ehcache) {
     } else {
       caches.remove(alias);
       value.setCache(null);
-      throw failure;
+      throw new RuntimeException("Cache '"+alias+"' creation in EhcacheManager failed.", failure);
```

Comment:
I would pick an `IllegalStateException` here rather than the generic `RuntimeException`.